src/main/java/com/datasonnet/spi/DataFormatService.java

```diff
@@ -1,16 +1,14 @@
-package com.datasonnet.portx.spi;
+package com.datasonnet.spi;
 
 import java.util.*;
 
 public class DataFormatService {
 
     private static DataFormatService service;
 
-    private static Map<String, List<DataFormatPlugin>> pluginRegistry;
+    private Map<String, List<DataFormatPlugin>> pluginRegistry = new HashMap<>();
 
-    private DataFormatService() {
-        pluginRegistry = new HashMap<>();
-    }
+    DataFormatService() { }
```

Comment:
This makes it available to tests in the same package, but still unavailable to pretty much everything else